cmake/developer_package/compile_flags/sanitizer.cmake

```diff
@@ -38,17 +38,19 @@ endif()
 # common sanitizer options
 if (DEFINED SANITIZER_COMPILER_FLAGS)
     # ensure symbols are present
-    set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -g -fno-omit-frame-pointer")
-    if(NOT OV_COMPILER_IS_CLANG)
-        # GPU plugin tests compilation is slow with -fvar-tracking-assignments on GCC.
-        # Clang has no var-tracking-assignments.
-        set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -fno-var-tracking-assignments")
-    endif()
-    # prevent unloading libraries at runtime, so sanitizer can resolve their symbols
-    set(SANITIZER_LINKER_FLAGS "${SANITIZER_LINKER_FLAGS} -Wl,-z,nodelete")
-
-    if(OV_COMPILER_IS_CLANG AND NOT WIN32 AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 8.0)
-        set(SANITIZER_LINKER_FLAGS "${SANITIZER_LINKER_FLAGS} -fuse-ld=lld")
+    if (NOT WIN32)
+        set(SANITIZER_COMPILER_FLAGS "${SANITIZER_COMPILER_FLAGS} -g -fno-omit-frame-pointer")
```

Comment:
I belive you need "/Oy-" for windows https://docs.microsoft.com/en-us/cpp/build/reference/oy-frame-pointer-omission?view=msvc-170

Reply:
Added in last commit